corehq/apps/translations/tests/test_bulk_app_translation.py

```diff
@@ -805,6 +888,20 @@ def test_unchanged_upload(self):
         self.assert_question_label("question2", 0, 0, "en", "/data/foo/question2")
 
 
+class MovedModuleTest(BulkAppTranslationTestBaseWithApp):
+    """
+    Test the bulk app translation upload when the itext reference in a question
+    in the xform body does not match the question's id/path.
+
+    The upload is an unchanged download.
+    """
+    file_path = "data", "bulk_app_translation", "moved_module"
```

Comment:
Definitely feel free to call this out of scope, but given that testing a moved module *could* test a pretty basic app, what would you think of using `AppFactory` instead of an external `app.json`? The JSON-based tests are harder to debug and update.

Reply:
Another great idea!